### PR TITLE
Cult will no longer yank the shuttle if it's on the way to centcomm

### PR DESCRIPTION
**Pull request by ShiftyRail**
It will achieve minor victory instead.
See title. 
[gameplay]

## Reasoning

0. I'm part of the sec metaclub
1. Yanking a shuttle that is ALREADY on transit feels like a dick move, since there's no way to actually scout for a "sacrifice in progress" if it's on the way to centcomm.
2. I don't believe it's healthy to reward calling the shuttle instantly when cult is called out, out of fear that a later call will be very susceptible to shuttle being dragged back to the station with a much lower chance to defend it if you took a severe beating in trying to fight cult.
3. Other antags (notably, Nuke Ops and new malf) only have 5 minutes to act when the shuttle is called. Cultists get 15 minutes. This PR, by, the way, only removes their possibility to act for the last 2 minutes.

:cl:
- tweak: The shuttle can no longer be recalled when it is on its way to centcomm by bloodstones raising.

### CODE REVIEW

**Comment by iFlashYou** [comment]
Buff cult in return

**Comment by aceedex** [comment]
Tbh I feel it shouldnt auto recall after 5 minutes. An aspect of cult 2.0 that I miss is that if the trail of a cult goes cold you can force more obvious and pointed actions from them by putting them on the clock

**Comment by Irradiation** [comment]
I've said it on the discord, but whenever the shuttle arrives, it's ironically the best time to do culty shit because nobody will give a shit about trying to protect every crew member.

**Comment by Sooose** [comment]
Please don't remove this it is my favorite feature because it's hilarious.

**Comment by Allspice** [comment]
regarding 3:
1. Newcops can nick a comms chip and emag it on their chip to recall to their hearts' intent, if so inclined.
2. Malf originally had autorecall, but now it's sort of niggered in that it both cannot recall and the shuttle timer is frequently much earlier to conclude, what with the delta clock going off of byondseconds.

**Comment by Caboozles** [comment]
I'd be fine with a cult minor victory if they're such slow bastards as to only act when the shuttle is in transit.

**Comment by DeityLink** [comment]
>cult minor victory
>at Act 2

this is ridiculous

and I'll join the team of those who find this hilarious. It's a nice possibility of comeback that doesn't even happen that often.

Cult really doesn't need more nerfs. Barely 33% of cults reached Act 3 in the past week. The real issue of cult right now is new players drawing runes in plain sight.

0. Please stop forcing so many cult design decisions when you don't even play cultist. Can you imagine being game designer for a game that you don't play? I mean I'd legit welcome your suggestions if you actually took the effort of giving it a go in its current state.
1. Can be remedied. I've pondered about adding global sound effects to indicate that a sacrifice is beginning. So people get a few seconds before Act 3 begins. So that's a valid concern.
2. Ironically this PR will encourage crews to call the shuttle quickly upon cults being sighted before any fighting even occurs which is something I tried very hard to get rid of with cult 3.
3. Nuke Ops can still detonate the nuke while the shuttle is on its way back to centcom.

**Comment by Irradiation** [comment]
> Can you imagine being game designer for a game that you don't play?

I wish c*ders and c*llabs were stopped a few years back, because that applied perfectly to them.

**Comment by Caboozles** [comment]
I could give less of a shit about the nuke going off when the shuttle is in motion. It doesn’t turn the shuttle around and gib people.

If the shuttle docks, everyone’s set to leave. If it’s about to dock then whiplashes around, everyone has to then pile around and go hunt for these stones.

It’s a major pain.

**Comment by DeityLink** [comment]
You're not getting my point. It's all BUT a victory.

**Comment by bathosbathtime** [comment]
There's a huge difference between "a nuke is detonated in the last seconds of the round while on the station" and "central command sends its shuttle back mid-transit because of these funky stones"

**Comment by DeityLink** [comment]
"Centcom sends the crew back to fight and possibly prevent the rise of an evil Elder God" sounds plausible to me. The only funky bit if anything is that the shuttle comes back instantly but that was by design to not put the crew at a bigger disadvantage.

Also the nuke blowing up is "always" gonna happen in the last seconds of a round, duh.

**Comment by bathosbathtime** [comment]
It sounds stupid to me.  Has nar'sei risen before?  How do they know what the blood stones do and that they are even on station?  If it is dire as you say it is, why not automatically trigger a death squad as well?
It is behavior unique to cult, and it isn't intuitive or well thought out behavior at that.  It is weird from a gameplay dynamic and its stupid from a narrative standpoint.

**Comment by DeityLink** [comment]
>Has nar'sei risen before?

If you really care about lore then I'll tell you that yes. In my vision Nar-Sie rising only causes some sectors of space to disappear, preventing Nanotrasen to re-build a station there to harvest plasma, hence why they'd like to prevent that. Not that it should matter since I try to have as little of it actually in the game to allow people to keep their headcannons.

>How do they know what the blood stones do and that they are even on station?

In lovecraftian fiction, the protagonists often know a lot about cults from books and writings found before they even come (if at all) in contact with those. It's not a stretch to think that Nanotrasen has an occult research branch, especially when it sends command alerts titled "Occult Activity Detected"

>If it is dire as you say it is, why not automatically trigger a death squad as well?

That's already a more sensible idea. Admins often send strike teams during late cult rounds, and the crew itself often calls an ERT anyway.

>It is weird from a gameplay dynamic and its stupid from a narrative standpoint.

The opposite is true in my opinion.

**Comment by Irradiation** [comment]
The thing I find really wonky about the auto-recall thing is that 
>0:02 to CentComm
>INSTANTLY teleported to the station even though it takes 10 minutes for the shuttle to arrive at the station normally or 2 minutes to arrive to CentComm.
Those are some sick ass logistics and shuttles that can be hyper performant when they feel like it.

**Comment by DeityLink** [comment]
re-posting what I said 2 posts ago:

`The only funky bit if anything is that the shuttle comes back instantly but that was by design to not put the crew at a bigger disadvantage.`

**Comment by bathosbathtime** [comment]
>Anti-cult strike team getting automatically triggered
>
Now we're getting somewhere.  Still don't like the shuttle yank since it's jank but a strike-team auto-triggered on blood stones would make the situation far less surreal to me.

**Comment by DeityLink** [comment]
I made a design doc for an inquisition strike team at some point but after some thinking I judged that we shouldn't add even more new mechanics that people would need to learn just to deal with cult.

So yeah an ERT getting automatically triggered if there wasn't one already sounds like a much better compromise to me. Especially considering #22258

That'd fix two issues with one stone.

**Comment by ShiftyRail** [comment]
This PR is mostly symbolic because it only "takes" 2 minutes of time out of the 15 cultists have
It's very frustrating to have to stay behind and scout maint for that sacrifice when you're supposed to guard escape.

As for victory, a sacrifice happening means that cult made it to the "unrecallable" phase, which means they achieved 60% of their objectives

**Comment by DeityLink** [comment]
Still a crew victory, don't deny that.

**Comment by ghost** [comment]
Cultists should get a rune that takes comparable effort to a raise structure rune which would recall shuttle when completed, even if time to arrive is less than 5 minutes. It would have to be available pre-sacrifice though.

**Comment by SonixApache** [comment]
- who knows how many dead
- crew abandons station so cult can do whatever
- cult just has to spread some blood to pull ol' narnar over
- somehow not a cult victory

deity pls

**Comment by DeityLink** [comment]
The point of doing it aboard the station is so that the crew can be offered to Nar-Sie. If the crew escapes aboard the shuttle that'd be like welcoming you to your birthday party but your cake and presents are all gone. I think you'd be pretty pissed off.

**Comment by ShiftyRail** [comment]
Lore or not lore, this is about two minutes of time. I don't think it's so big of a deal.

**Comment by DeityLink** [comment]
Still a cult nerf.

**Comment by SirSkeleto** [comment]
imo the biggest problem isn't with balance with respect to the game modes, it's just kind of a "fuck you" to people who are ready to go home, especially when the cult has been super slow all round. People who are too scared to take risks or even god forbid go a bit loud shouldn't get the added advantage of being able to accomplish objectives after the metaphorical train has literally left the station, and being able to win the round on top of that.

**Comment by DeityLink** [comment]
If the cult was this bad, they won't be able to perform the sacrifice while the shuttle is on its way home. It'll become more of an issue against crews trying to call the shuttle early against a competent cult.

**Comment by agentofzog2** [comment]
Cult should be able to keep all their crutches until it gets to the point it can actually sometimes win.

**Comment by SonixApache** [comment]
they threw the party, so at least it's something
a minor victory if you will

**Comment by DeityLink** [comment]
@agentofzog2 please elaborate.

@SonixApache do you you seriously believe for one second that such a victory would feel remotely satisfying at all for cultists? I can assure you though that the crew would feel smug as fuck. It's a crew minor victory if anything.

**Comment by DeityLink** [comment]
All this discussion made me figure out a potential exploit #22474 

With that fixed, the cult would have to invade and successfully kidnap the target FROM the moving shuttle, which in my opinion should allow the round to continue. @ShiftyRail

**Comment by SonixApache** [comment]
gonfligds :D

**Comment by DeityLink** [comment]
Now that #22474 got merged, I suggest we observe for a while how often this actually happens anymore before nerfing it.

**Comment by ShiftyRail** [comment]
I still believe that this should not happen on principle alone.

**Comment by Kurfursten** [comment]
We have had 2 weeks of observation time, what is the status?

**Comment by ShiftyRail** [comment]
For me this is beyond empirical concerns and I just hate it because it fucks with the "shuttle has left, you can begin to relax" principle of ss13.

Having to do the bloodstone search & destroy game just after I thought I was done for the round is terribly demotivating.

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 1]
```diff
@@ -371,6 +376,16 @@ var/veil_thickness = CULT_PROLOGUE
 	for (var/reminder in cult_reminders)
 		R.antag.store_memory("Cult reminder: [reminder].")
 
+/datum/faction/bloodcult/proc/minor_victory()
+	for(var/datum/role/cultist/C in members)
+		var/mob/M = C.antag.current
+		if (M && iscultist(M))
+			to_chat(M,"<span class='sinister'>While the sacrifice was correctly completed, we were not fast enough to prevent our ennemies from feeling.</span>")
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm]: >our ennemies from feeling

Nar-Sie is trying to take your feels

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 2]
```diff
@@ -217,6 +217,10 @@ var/veil_thickness = CULT_PROLOGUE
 	initialize_cultwords()
 	AppendObjective(/datum/objective/bloodcult_reunion)
 
+
+/datum/faction/bloodcult/minorVictoryText()
+	return "The cult did not manage to summon Nar-Sie, but could achieve most of its objectives and sacrifice the designated victim."
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm]: Better phrasing would be "The cult completed its sacrificial ritual, but not in time to summon Nar-Sie."

**Comment by DeityLink** [comment]
For me it's a nerf to cultists boarding the shuttle to try and contest the sacrifice target. An unnecessary punishment, and I'm clearly not alone with that opinion.

@Kurfursten It's such a rare occurrence it probably hasn't happened recently. It's even less likely to occur now that #22474 got merged. But should it happen, the cult should be rewarded for their trouble.

Oh and finally please let's not add "minor victories" to cult.

**Comment by ShiftyRail** [comment]
Someone else than me either close this or make it through a server poll because I'm **not** compromising on it. This is already a compromise ; cultists should get fucked if the shuttle docked. Like other antags are.

**Pull request merged**